templates/create-daml-app/ui/src/components/LoginScreen.tsx.template

```diff
@@ -100,7 +113,8 @@ const LoginScreen: React.FC<Props> = ({onLogin}) => {
       <DamlHubLoginBtn
         onLogin={creds => {
           if (creds) {
-            login({party:creds.party, user: {userId: creds.partyName, primaryParty: creds.party}, token:creds.token});
+            // TODO (MK) Fix public party in Daml hub
```

Comment:
both of these todos were already broken before, just breaking them slightly more. working on the daml hub one, #12849 tracks the auth0 part which the runtime team will work on.